staking/programs/wallet-tester/output.rs

```diff
@@ -0,0 +1,774 @@
+#![feature(prelude_import)]
```

Comment:
does this need to be checked in? it looks autogenned

Reply:
Thanks for catching this